src/market.ts

```diff
@@ -616,8 +620,24 @@ function handleAccountPositionProcessed(
   liquidationFee: BigInt,
   subtractiveFees: BigInt,
 ): void {
+  // TODO: should we trap these and avoid creating the MarketAccount?
```

Comment:
@arjun-io Thoughts on this?  The following query shows we're creating empty accumulations for them:
```
query EmptyAccounts {
  marketAccounts(where:{account:"0x0000000000000000000000000000000000000000"}) {
    account { id }
    positions { id }
    maker
    long
    short
    collateral
    accumulators { 
      id 
      collateral
      fees
    }
  }
}
```